dist/appForms-backbone.js

```diff
@@ -4430,12 +4431,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
+    if (displayIndex !== 0 || hasCustomPageName) {
```

Comment:
Sorry about this, but I rebased this prematurely. I've tried to fix this issue [here](https://github.com/feedhenry/fh-js-sdk/pull/164/commits/46db12691b423676ba9348a294d6690674d0e0f3#diff-c6c680480ef790b1aa733d3fd485e464R76) and also added an additional [test](https://github.com/feedhenry/fh-js-sdk/pull/164/commits/46db12691b423676ba9348a294d6690674d0e0f3#diff-7e2845e23b8af8e0cecc634dcf3e8e20R21) to verify. Was this what you had in mind?